screens/Menu/NearbyInterest.js

```diff
@@ -50,8 +53,19 @@ function NearbyInterest(props) {
      * The method will let us navigate to the NearbyInterestDetails screen
      * @param  {} =>{props.navigation.navigate("NearbyInterestDetails)"
      */
-    const goToNearbyInterestDetails = () => {
-        props.navigation.navigate("NearbyInterestDetails");
+    const goToNearbyInterestDetails = (item) => {
+        props.navigation.navigate("NearbyInterestDetails", {
```

Comment:
**Suggestion:** Create an object that containts all those attributes and only pass that object. That object could be a state hook. 
Check `PreviewDirections` and `Directions` for similar implementation.

Reply:
Got it!